src/services/auth/signIn.js

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+
+export const signIn = async ({
+	loginData,
+	dispatch,
+	initialFormState,
+	toggleAuth,
+	toggleShow,
+	setStatusMsg,
+	rememberMe,
+	toggleLoader,
+}) => {
+	try {
+		toggleLoader();
+
+		const res = await axios.post("/api/auth/login", loginData);
+		if (res.status === 200) {
+			localStorage.setItem("noteToken", res.data.encodedToken);
+			toggleAuth();
```

Comment:
You could have destructured it like this.
Instead of `const res => const {data: {foundUser}}`